molecule/README.md

```diff
@@ -0,0 +1,42 @@
+Molecule tests for the role.
+
+# Test Matrix
+
+Test options "flow down" thro table unless changed.
+
+test   | # partitions | # groups in partitions? | other
+---    | ---          | ---                     | ---
+test1  | 1            | N                       | 2x compute node, sequential names (default test)
+test1b | 1            | N                       | 1x compute node
+test1c | 1            | N                       | 2x compute nodes, nonsequential names
+test2  | 2            | N                       | 4x compute node, sequential names
+test3  | 1            | Y                       |
+
+# Local Installation & Running
+
+Local installation on a Centos7 machine looks like:
+
+    sudo yum install -y gcc python3-pip python3-devel openssl-devel python3-libselinux
+    sudo yum install -y docker-ce docker-ce-cli containerd.io
+    sudo yum install -y yum-utils
+    sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+    sudo yum install -y docker-ce docker-ce-cli containerd.io
+    python3 -m pip install --upgrade --user pip
+    python3 -m pip install --upgrade --user setuptools
+    python3 -m pip install --upgrade --user "molecule[lint]"
```

Comment:
wait, why not just pip3 install?

Reply:
I just copied the molecule docs here. Although I'd note a core python dev [suggests ](https://snarky.ca/why-you-should-use-python-m-pip/) always using `python -m pip` anyway.

Hopefully the local install instructions won't be that relevant - I'm working on a travis setup for molecule tests anyway.

Reply:
Fixed now by commit.

Reply:
I understand why `python -m pip` is better practice :) I now need to just remember to use it.